tests/unit/type/stream-cgroups.tcl

```diff
@@ -509,6 +537,39 @@ start_server {
         assert {[llength $reply] == 0}
     }
 
+    test {XCLAIM with MKGROUP option to create consumer group} {
+        # Add 3 items into the stream, and create a consumer group
+        r del mystream
+        set id1 [r XADD mystream * a 1]
+        set id2 [r XADD mystream * b 2]
+        set id3 [r XADD mystream * c 3]
+
+        after 200
```

Comment:
why do we need all these sleeps?

Reply:
I will address your comments in next a few days, sorry for that.

Reply:
updated @oranagra